content/en/security/cloud_siem/investigate_a_security_signal.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Investigate a Security Signal
+kind: documentation
+disable_toc: false
+further_reading:
+  - link: "/cloud_siem/detection_rules/"
+    tag: "Documentation"
+    text: "Learn about the conditional logic of detection rules"
+  - link: "https://www.datadoghq.com/blog/monitor-1password-datadog-cloud-siem/"
+    tag: "Blog"
+    text: "Monitor 1Password with Datadog Cloud SIEM"
+---
+
+## Overview
+
+A Cloud SIEM security signal is created when Datadog detects a threat while analyzing logs against detection rules. View, search, filter, and correlate security signals in the Signal Explorer without needing to learn a dedicated query language. You can also assign security signals to yourself or another user in the Datadog platform. In addition to the Signal Explorer, you can configure [Notification Rules][1] to send signals to specific individuals or teams to keep them informed of issues.
+
+You must have the `Security Signals Write` permission to modify a security signal, such as change the state and view signal action history in [Audit Trail][2]. See [Role Based Access Control][3] for more information about Datadog's default roles and granular role-based access control permissions available for Datadog Security in the Cloud Security.
```

Comment:
Leaving this here for now. The plan is to pull this out and have a general RBAC doc for all security products.